### DISCUSSION TITLE
[分享]神經網路知識圖形化教學網站

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#534] [Ideas]
### TenforFlow Playground
TensorFlow Playground，是一個用於可視化簡單神經網路教學和實驗的在線平臺。它非常強大且易於使用。如果您已經具有一些神經網絡知識，您可以在這個超級易懂的演示中快速體驗一個演算法工程師的調參工作。您可以在本地執行此項目，前往項目github地址，將其本地clone，並按順序執行npm i、npm run build、npm run serve。

Github:https://github.com/tensorflow/playground
工具:https://developers.google.com/machine-learning/crash-course/introduction-to-neural-networks/playground-exercises
教學:
(1)https://ithelp.ithome.com.tw/articles/10218185
(2)https://github.com/leemengtaiwan/deep-learning-resources

**Comment by limitx0** [comment]
GitHub的那個教學好讚
很完整也很詳細

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Yilin20021016** [comment]
這個東西內容很多很有趣 但對我來說有點太難了

**Comment by bww0401** [comment]
把neural network變成可視化圖像之後，感覺更方便學習，如果系上的教授也都這樣教就好了: )
